src/Compilers/VisualBasic/Portable/Symbols/Source/SynthesizedFieldSymbol.vb

```diff
@@ -59,6 +59,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 Return
             End If
 
+            Dim compilation = Me.DeclaringCompilation
+            If Type.ContainsTupleNames() AndAlso
+                compilation.HasTupleNamesAttributes AndAlso
```

Comment:
turns out checking whether an attribute is needed syntactically is a nontrivial change. So I decided to do it as a separate workitem, including the diagnostics for explicit use of tuple names attributes

---
In reply to: [79716733](https://github.com/dotnet/roslyn/pull/13923#discussion_r79716733) [](ancestors = 79716733,79676084)